REFERENCE.md

```diff
@@ -339,6 +339,14 @@ Determins if a tls key is generated
 
 Default value: `false`
```

Comment:
This file is auto generated as part of the release process. Do not think it should be in the pull request.

Reply:
@bastelfreak said:
> Can you also document this feature in the README.md?
https://github.com/voxpupuli/puppet-openvpn/pull/339#issuecomment-495994855

I assumed this was what he meant. I can remove it if you want, but it would be nice to get clarification.

Reply:
Also, the [version hosted on puppet forge](https://forge.puppet.com/puppet/openvpn/reference) is outdated, so I am guessing the file hasn't actually been updated as part of the release process. For example, https://github.com/voxpupuli/puppet-openvpn/pull/339/commits/61fefebd11b2339a72fc3dd3d263be418e74a3d4#diff-eb120e74eba90da6132666ceb0405bf9R386 - `compression` was not an Enum in the last released version.